testData/ontologies/flat.json5

```diff
@@ -306,6 +306,13 @@
             "type": "boolean",
             "default": "false"
         },
+        "http://registry.gpii.net/common/supportTool": {
+            "title": "Support Tools",
+            "description": "Whether to enable/disable certain support tools",
+            "type": "array",
+            "default": "dictionary", // What if there's no default value?
```

Comment:
I would imagine that this should be: `"default": [ "dictionary" ]` (or `[]` for no support tool by default) since the type is an array